packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -950,6 +952,21 @@ export class DeltaManager
 
     private readonly opHandler = (documentId: string, messagesArg: ISequencedDocumentMessage[]) => {
         const messages = messagesArg instanceof Array ? messagesArg : [messagesArg];
+        const sequence = messages[0].sequenceNumber;
```

Comment:
@GaryWilber - I'm thinking to remove this code / check. I think it served its purpose (being directly and simple way of validating claim in stress tests). While not direct, the check in enqueueMessages will capture same issue (not 100% of the time, but most of the time), and will likely be sufficient going forward. Thoughts?

Reply:
Basically iteration before did not give me 100% confidence in the claim, but this addition (below in same PR) will for future runs:

this.prevEnqueueMessagesReason = this.pending.length > 0 ? "unknown" : reason;